src/main/java/ch/kalunight/zoe/repositories/TeamRepository.java

```diff
@@ -0,0 +1,168 @@
+  }
+  
+  public static List<Team> getAllPlayerInTeams(long guildId, String language) throws SQLException {
+    List<DTO.Player> playerWithNoTeam = new ArrayList<>();
+    List<DTO.Player> players = PlayerRepository.getPlayers(guildId);
+    List<DTO.Team> teamsDto = TeamRepository.getTeamsByGuild(guildId);
+    List<Team> teams = new ArrayList<>();
+    for(DTO.Player player : players) {
+      player.leagueAccounts = LeagueAccountRepository.getLeaguesAccounts(guildId, player.player_discordId);
+    }
+    
+    List<DTO.Player> playerWithATeam = new ArrayList<>();
+    
+    for(DTO.Team team : teamsDto) {
+      Team teamToAdd = new Team(team.team_name);
+      teams.add(teamToAdd);
+      for(DTO.Player player : players) {
+        if(team.team_id == player.player_fk_team) {
+          teamToAdd.getPlayers().add(player);
+          playerWithATeam.add(player);
+        }
+      }
+    }
+    
+    for(DTO.Player player : playerWithATeam) {
+      players.remove(player);
+    }
+    
+    playerWithNoTeam.addAll(players);
+
+    List<Team> allTeams = new ArrayList<>();
+    allTeams.addAll(teams);
+    if(!playerWithNoTeam.isEmpty()) {
+      allTeams.add(new Team(LanguageManager.getText(language, "teamNameOfPlayerWithoutTeam"), playerWithNoTeam));
+    }
+
+    return allTeams;
+  }
+  
+  public static DTO.Team getTeamByPlayerAndGuild(long guildId, long discordId) throws SQLException {
+    ResultSet result = null;
+    try (Connection conn = RepoRessources.getConnection();
+        Statement query = conn.createStatement(ResultSet.TYPE_SCROLL_INSENSITIVE, ResultSet.CONCUR_READ_ONLY);) {
+      
+      String finalQuery = String.format(SELECT_TEAM_BY_SERVER_AND_DISCORD_ID, discordId, guildId);
+      result = query.executeQuery(finalQuery);
+      int rowCount = result.last() ? result.getRow() : 0;
+      if(rowCount == 0) {
+        return null;
+      }
+      return new DTO.Team(result);
+    }finally {
+      RepoRessources.closeResultSet(result);
+    }
+  }
+  
+  public static List<DTO.Team> getTeamsByGuild(long guildId) throws SQLException {
+    ResultSet result = null;
+    try (Connection conn = RepoRessources.getConnection();
+        Statement query = conn.createStatement(ResultSet.TYPE_SCROLL_INSENSITIVE, ResultSet.CONCUR_READ_ONLY);) {
+      
+      String finalQuery = String.format(SELECT_TEAMS_BY_GUILD_ID, guildId);
+      result = query.executeQuery(finalQuery);
+      
+      List<DTO.Team> teams = Collections.synchronizedList(new ArrayList<>());
+      int rowCount = result.last() ? result.getRow() : 0;
+      if(rowCount == 0) {
+        return teams;
+      }
+      result.first();
+      while(!result.isAfterLast()) {
+        teams.add(new DTO.Team(result));
+        result.next();
+      }
+      
+      return teams;
+    }finally {
+      RepoRessources.closeResultSet(result);
+    }
+  }
+  
+  public static DTO.Team getTeam(long guildId, String teamName) throws SQLException {
+    ResultSet result = null;

```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Always check the return of one of the navigation method (next,previous,first,last) of a ResultSet.](https://app.codacy.com/manual/KaluNight/Zoe-Discord-Bot/pullRequest?prid=4739924)